primitives/phragmen/compact/src/lib.rs

```diff
@@ -181,6 +198,28 @@ fn struct_def(
 		{
 			const LIMIT: usize = #count;
 		}
+
+		impl<#voter_type, #target_type, #weight_type> #ident<#voter_type, #target_type, #weight_type> {
+			/// Get the length of all the assignments that this type is encoding. This is basically
+			/// the same as the number of assignments, or the number of voters in total.
+			pub fn len(&self) -> usize {
+				let mut all_len = 0usize;
+				#len_impl
+				all_len
+			}
+
+			/// Get the total count of edges.
+			pub fn edge_count(&self) -> usize {
+				let mut all_edges = 0usize;
+				#edge_count_impl
+				all_edges
+			}
+
+			/// Get the average edge count.
+			pub fn average_edge_count(&self) -> usize {
```

Comment:
I think this function is not used anymore

Reply:
I would keep this around preferably, I think it will be helpful. I think a release build will strip it off anyhow right?

Reply:
yes I think so, I'm ok to keep it